sepolicy/bkmgrd.te

```diff
@@ -0,0 +1,11 @@
+type bkmgrd, domain;
```

Comment:
We are using permissive,so leave sepolicies as it is. All sepolicies would be defined after ROM is booted